web/src/bootstrap_typeahead.ts

```diff
@@ -173,17 +167,6 @@ import * as tippy from "tippy.js";
 import * as scroll_util from "./scroll_util";
 import {get_string_diff, the} from "./util";
 
-function get_pseudo_keycode(
-    event: JQuery.KeyDownEvent | JQuery.KeyUpEvent | JQuery.KeyPressEvent,
-): number {
-    const isComposing = event.originalEvent?.isComposing ?? false;
-    /* We treat IME compose enter keypresses as a separate -13 key. */
```

Comment:
Why is it safe to remove this special case?

Reply:
See https://github.com/zulip/zulip/pull/31427#discussion_r1727502797